robotpy_ext/autonomous/selector.py

```diff
@@ -236,7 +237,11 @@ def run(self, control_loop_wait_time=0.020, iter_fn=None, on_exception=None):
             except:
                 on_exception()
             
-            iter_fn()
+            if isinstance(iter_fn, Iterable):
```

Comment:
Not sure I'm a huge fan of this check. If we want to do this here, we should check for list or (list, tuple).

Reply:
👍 I was sure there was a better way but google got the best of me